controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStream.java

```diff
@@ -216,7 +216,7 @@ private String getWritersTableName(String id) {
                                                                   OperationContext context,
                                                                   Map<String, TxnWriterMark> writerMarks) {
         Preconditions.checkNotNull(context, "operation context cannot be null");
-
+        log.debug("completeCommittingTransactions.");
```

Comment:
this log line can have more context.

Reply:
Didn;t intend to add this log , had added while debugging, got left out by mistake... will remove.

Reply:
fixed

Reply:
ok